File: Python/tests/test_prl_fictitious.py
import pytest

from hbayesdm.models import prl_fictitious


def test_prl_fictitious():
    _ = prl_fictitious(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_fictitious.R
context("Test prl_fictitious")
library(hBayesDM)

test_that("Test prl_fictitious", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_fictitious(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
